src/backend/marsha/core/tests/test_xapi.py

```diff
@@ -185,23 +188,90 @@ def test_xapi_statement_live_video(self):
             live_type=RAW,
         )
 
-        mock_token_user = mock.MagicMock()
-        mock_token = mock.MagicMock()
-        type(mock_token).payload = mock.PropertyMock(
-            return_value={
-                "user": {
-                    "id": "foo",
+        jwt_token = AccessToken()
+        jwt_token.payload["user"] = {"id": "foo"}
+        jwt_token.payload["context_id"] = "course-v1:ufr+mathematics+0001"
+
```

Comment:
Minor but I would add a session_id different in the JWT token, to do a test that is more accurate, showing that even with session_id existing, it's still a user that is chosen.

Reply:
:+1: I will also real UUID and user identifier format.